pkg/helm/actions/config.go

```diff
@@ -35,6 +41,10 @@ func GetActionConfigurations(host, ns, token string, transport *http.RoundTrippe
 		},
 		Transport: transport,
 	}
+	inClusterCfg, err := rest.InClusterConfig()
+	if err == nil {
```

Comment:
What if:

```golang 
err != nil {
   // handle this?
}
```
You will typically see the `err != nil` pattern in golang.  The func should return an error as a second return value and make sure it can be handled.

Reply:
well, in this case, `err == nil` means that we are running in-cluster, and hence we can use the discovered CA file in helm context as well. Running console outside of cluster happens in dev mode.

Reply:
Just a log message should be fine then, in case this would happen for unexpected reasons.

Reply:
fixed.